pkg/polaris/graphql/aws/cloud.go

```diff
@@ -101,14 +102,15 @@ func (a API) CloudAccountWithFeatures(ctx context.Context, id uuid.UUID, feature
 // CloudAccountsWithFeatures returns the cloud accounts matching the specified
 // filter. The filter can be used to search for AWS account id, account name
 // and role arn.
-func (a API) CloudAccountsWithFeatures(ctx context.Context, feature core.Feature, filter string) ([]CloudAccountWithFeatures, error) {
+func (a API) CloudAccountsWithFeatures(ctx context.Context, feature core.Feature, filter string, statusFilters []core.Status) ([]CloudAccountWithFeatures, error) {
```

Comment:
Maybe we can add this piece of code after the `log.Print` statement:
```
if statusFilters == nil {
    statusFilters = []core.Status{}
}
```
This would make it safe to pass in `nil` as a value for the `statusFilter` slice.